packages/plugin-node-device/device.js

```diff
@@ -5,7 +5,7 @@ const { isoDate } = require('@bugsnag/core/lib/es-utils')
  */
 module.exports = {
   init: (client) => {
-    const device = { hostname: client.config.hostname }
+    const device = { hostname: client.config.hostname, versions: process.versions }
```

Comment:
This should be added under `metadata.device` as it's not a documented `device` field in the API.